WordPress/src/main/java/org/wordpress/android/ui/compose/components/menu/dropdown/JetpackDropdownMenu.kt

```diff
@@ -60,10 +64,17 @@ fun JetpackDropdownMenu(
                 isMenuVisible = !isMenuVisible
             }
         )
+        val cascadeMenuWidth = 200.dp
         CascadeDropdownMenu(
-            modifier = Modifier.background(MenuColors.itemBackgroundColor()),
+            modifier = Modifier
+                .background(MenuColors.itemBackgroundColor())
+                .width(cascadeMenuWidth),
             expanded = isMenuVisible,
             onDismissRequest = { isMenuVisible = false },
+            offset = if (LocalLayoutDirection.current == LayoutDirection.Rtl) DpOffset(
+                cascadeMenuWidth,
+                0.dp
+            ) else DpOffset.Zero,
```

Comment:
❤️  Nice solution! Thanks!

🔍  Nitpick: when an if statement spawns multiple lines I would expect the usage of braces `{ }`. Maybe you can fit a 1-line `if-else` clause by writing it like this:

```kotlin
offset = DpOffset(
    if (LocalLayoutDirection.current == LayoutDirection.Rtl) cascadeMnuWidth else 0.dp,
    0.dp,
)
```

Reply:
Done 👍